app/src/main/kotlin/compiler/intermediate_form/Addressing.kt

```diff
@@ -1,6 +1,7 @@
 package compiler.intermediate_form
 
 typealias MemoryAddress = ULong
+const val memoryUnitSize: ULong = 8u
 
 sealed class Addressing {
```

Comment:
Yup, we've decided, that we'd only need `MemoryLabel` IFTNode. Removed this file totally and moved `memoryUnitSize` to `DefaultFunctionDetailsGenerator.kt`

Reply:
It actually turns out that while we don't need it here, another PR (which adds assembly instruction set) uses this class (with some modifications), so the file is going to be reintroduced there. Still, it can be removed here.